tests/test_new_tickets.py

```diff
@@ -1,29 +0,0 @@
-"""
-Improved tests for ep2019+ based on the new cart implementation
-"""
-import pytest
-
-
-@pytest.mark.xfail
-def test_user_can_configure_their_ticket(db):
```

Comment:
where did those go? :)

Reply:
I realised that those already existed in `test_user_panel`, so there was no need to implement them :) E.g. for this one: https://github.com/EuroPython/epcon/blob/dev/ep2020/tests/test_user_panel.py#L84